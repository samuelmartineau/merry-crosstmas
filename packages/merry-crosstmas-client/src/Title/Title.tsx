import React from 'react';
import { withStyles } from '@material-ui/core/styles';

const styles = theme => ({
  title: {
    marginTop: '1em',
    textAlign: 'center',
    fontSize: '1.5rem',
    textShadow: '1px 1px white',
    padding: '0 .5em',
  },
});

const Title = ({ classes, children }) => {
  return <h2 className={classes.title}>{children}</h2>;
};

export default withStyles(styles)(Title);
