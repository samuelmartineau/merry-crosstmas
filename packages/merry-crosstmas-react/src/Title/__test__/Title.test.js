import React from "react";
import renderer from "react-test-renderer";
import Title from "../Title";

test("Title renders correctly", () => {
  const tree = renderer.create(<Title />).toJSON();
  expect(tree).toMatchSnapshot();
});
test("Title renders correctly whith content", () => {
  const tree = renderer.create(<Title>Children test</Title>).toJSON();
  expect(tree).toMatchSnapshot();
});
