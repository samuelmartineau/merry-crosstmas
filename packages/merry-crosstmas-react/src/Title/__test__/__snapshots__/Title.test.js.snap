// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<h2
  className="Title-title-1"
/>
`;

exports[`Title renders correctly whith content 1`] = `
<h2
  className="Title-title-1"
>
  Children test
</h2>
`;
